src/components/transport_manager/include/transport_manager/transport_manager_default.h

```diff
@@ -56,7 +52,12 @@ class TransportManagerDefault : public TransportManagerImpl {
    *
    * @return Code error.
    */
-  int Init(resumption::LastState& last_state) OVERRIDE;
+  int Init(resumption::LastStateWrapperPtr last_state_wrapper) OVERRIDE;
+
+  DEPRECATED
+  int Init(resumption::LastState&) {
+    return 1;
```

Comment:
needs to be defined properly, the function needs to work

Reply:
@jacobkeeler Done in 4fdae6f97e8445b4aebe5269c4163e028115f508

Reply:
@jacobkeeler Please take a look https://github.com/smartdevicelink/sdl_core/pull/2626#discussion_r373481596